ci/environment-upstream-dev.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
+  - cf_xarray
```

Comment:
how about installing this from source in `environment-upstream-dev.yml` and using the released version in `environmnent-dev.yml`?